assembl/templates/views/messageColumn.tmpl

```diff
@@ -0,0 +1,42 @@
+
+<div class="real-time-updates tac hidden phl">
+    <button class="btn-real-time js_loadPendingMessages">
+        <span class="pendingMessage"></span>
+    </button>
+</div>
+
+<style type="text/css">
+  #message_column_<%= column.getNumericId() %> .js_agentName,
```

Comment:
wow, this seems to be related to my theme architecture question. I'm really not sure we want to go into this way, it's too different from our current standards (imbricating CSS directly in templates of components is more React-y I guess)